packages/wonder-blocks-tooltip/util/is-obscured.test.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import {mount} from "enzyme";
+import isObscured from "./is-obscured.js";
+
+describe("isObscured", () => {
+    test("element is null, throws", () => {
+        // Arrange
+        const element = ((null: any): Element);
+
+        // Act
+        const underTest = () => isObscured(element);
+
+        // Assert
+        expect(underTest).toThrowError();
+    });
+
+    test("element is not obscured by anything, returns false", (done) => {
+        const arrange = (actAndAssert) => {
+            // Arrange
+            const nodes = (
+                <div>
+                    <div ref={actAndAssert}>Unobscured</div>
+                </div>
+            );
+            mount(nodes);
+        };
+
+        const actAndAssert = (ref) => {
+            const element = ((ReactDOM.findDOMNode(ref): any): Element);
+            // When not obscurred, elementFromPoint should return the element.
+            // Flow doesn't like us doing this to the document $FlowFixMe
+            document.elementFromPoint = jest.fn().mockReturnValue(element);
```

Comment:
Does this resent `elementFromPoint` after the test?  Not a big deal if it doesn't... I'm curious more than anything.

Reply:
Hmm, I think so, but I should check!

Reply:
I've added some code to reset it, just in case.